### PR TITLE
Efm32integration

### PR DESCRIPTION
**Pull request by LaplaceG**
The JS interpreter is now running on the EFM32 and working with the Espruino web IDE.
Use PD0/PD1 for TX/RX (Pin 4/6 on the expansion header on the EFM32GG starter kit).

No advanced hardware functions implemented yet.

### CODE REVIEW

**Comment by gfwilliams** [comment]
This is great - thanks for making this such a clean pull request!

File: f4tt5ylp.wp4.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Merge of espruino/master into efm32integration
```
Comment (tve) [inline f4tt5ylp.wp4.txt]: what's the purpose of this file?

Comment (gfwilliams) [inline f4tt5ylp.wp4.txt]: Good spot - looks like it's a temporary file created by Git (I think so you can edit the commit message). Not sure how it happened - maybe if you do a `git commit` in one window and edit the message, then do it again in a second window.

Comment (LaplaceG) [inline f4tt5ylp.wp4.txt]: Yeah, this is something git created. Should be removed.

Comment (gfwilliams) [inline f4tt5ylp.wp4.txt]: Just done.

**Pull request merged**